x/blob/client/cli/payforblob.go

```diff
@@ -31,19 +32,32 @@ const (
 
 func CmdPayForBlob() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "PayForBlobs [hexNamespaceID] [hexBlob]",
-		Short: "Pay for a data blob to be published to the Celestia blockchain",
-		Long: "Pay for a data blob to be published to the Celestia blockchain. " +
-			"[hexNamespaceID] must be a 10 byte hex encoded namespace ID, " +
-			"(for the namespace version 0). " +
-			"[hexBlob] can be an arbitrary length hex encoded data blob. " +
-			"Hex strings should not include a 0x prefix. " +
-			"This command only supports a single blob per invocation. ",
-		Args: cobra.ExactArgs(2),
+		Use: "PayForBlobs namespaceID blob",
+		// This example command can be run in a new terminal after running single-node.sh
+		Example: "celestia-appd tx blob PayForBlobs 0x00010203040506070809 0x48656c6c6f2c20576f726c6421 \\\n" +
+			"\t--chain-id private \\\n" +
+			"\t--from validator \\\n" +
+			"\t--keyring-backend test \\\n" +
+			"\t--fees 21000utia \\\n" +
+			"\t--yes",
+		Short: "Pay for a data blob to be published to Celestia.",
+		Long: "Pay for a data blob to be published to Celestia.\n" +
+			"namespaceID is the user-specifiable portion of a version 0 namespace. It must be a hex encoded string of 10 bytes.\n" +
+			"blob must be a hex encoded string of any length.\n" +
```

Comment:
[Optional-Sugestion]
I find the following portion of the previous version easier to interpret i.e. using the same argument names as in the short description, 
```suggestion
			"[hexNamespaceID] is the user-specifiable portion of a version 0 namespace. It must be a hex encoded string of 10 bytes.\n" +
			"[hexBlob] must be a hex encoded string of any length.\n" +
```

Reply:
The short description is "Pay for a data blob to be published to Celestia." Before and after this PR it didn't use the prefix `hex`. I removed the hex prefix b/c other arguments that are hex encoded don't explicitly state `hex` in the argument name. E.g. 

```
  celestia-appd tx qgb register [valAddress] [evmAddress] [flags]
```

An Ethereum address is a 42-character hexadecimal address